lm_eval/api/samplers.py

```diff
@@ -32,7 +32,8 @@ def get_context(self, doc, num_fewshot):
         labeled_examples = (
             self.delimiter.join(
                 [
-                    self.task.doc_to_text(doc) + self.task.doc_to_target(doc)
+                    # TODO: is separating doc_to_text and doc_to_target by one space always desired?
+                    self.task.doc_to_text(doc) + " " + self.task.doc_to_target(doc)
```

Comment:
@haileyschoelkopf I'm not sure with this yet, still leaning towards the previous version and have users manually add the space in the `doc_to_target`? But this does allow users to not think about it. But we don't know everyone wants to use spaces?

Reply:
I think the idea of an `example_delimiter` config is a fair middle ground.